app/views/settings/applications/_fields.html.haml

```diff
@@ -7,7 +7,7 @@
 .fields-group
   = f.input :redirect_uri, wrapper: :with_block_label, label: t('activerecord.attributes.doorkeeper/application.redirect_uri'), hint: t('doorkeeper.applications.help.redirect_uri')
 
-  %p.hint= t('doorkeeper.applications.help.native_redirect_uri', native_redirect_uri: Doorkeeper.configuration.native_redirect_uri)
+  %p.hint= t('doorkeeper.applications.help.native_redirect_uri', native_redirect_uri: "<code>#{Doorkeeper.configuration.native_redirect_uri}</code>".html_safe)
```

Comment:
Unfortunately, it doesn't work that smart. The escaping happens on the outermost level. The simpliest thing is just adding `.html_safe` to the `t` call. Also it would be better to use `content_tag(:code, Doorkeeper.configuration.native_redirect_uri)` instead of writing out the HTML.

Reply:
The alternative is renaming the string to `native_redirect_uri_html` which will skip escaping it, but that would require updating all locales.

Reply:
Okay, thanks for telling. I will try the first method and corresponding fixes a little later then.

Reply:
Went through nine circles of hell to deal with and install this heckin' Ruby! Took me no less than 1.5 hours. Just to install testing Mastodon instance because I'm tired of these blind changes and I guess you are tired of seeing these too.

Your method actually working:
![Screenshot](https://user-images.githubusercontent.com/10401817/70400945-f867f800-1a5f-11ea-9f34-fa4679d840f1.png)

*btw I also got more context as translator. That was effective :pray:*